### ISSUE TITLE
A11Y: Typeahead is missing aria-autocomplete and aria-expanded

### ISSUE THREAD

**Issue by ashpc** [carbon-design-system/carbon-addons-beta-react#22]
## Detailed description

Describe in detail the issue you're having. Is this a feature request (new component, new icon), a bug, or a general issue?

The Typeahead is missing aria-autocomplete and aria-expanded.

> Is this issue related to a specific component?

Typeahead

> What did you expect to happen? What happened instead? What would you like to see changed?

Typeahead should have all the aria values assigned so the there are no A11Y errors.

> What browser are you working in?

Chrome

> What version of the Carbon Design System are you using?

    "carbon-addons-beta-react": "1.3.2",

## Steps to reproduce the issue

1.  Use Typeahead and it will have errors related to missing aria-autocomplete and aria-expanded

## Add labels
Please choose the appropriate label(s) from our existing label list to ensure that your issue is properly categorized. This will help us to better understand and address your issue.
bug

**Comment by marijohannessen** [comment]
Hi @ashpc . This `TypeAhead` component was released as a beta component only, and has been replaced with the official component `MultiSelect` (found here https://github.com/carbon-design-system/carbon-components-react/tree/master/src/components/MultiSelect). This component is not to be used by any production teams so I am closing this issue. Thank you!
